Not really sure how to fix linting errors from Travis:
```
./homeassistant/components/sensor/discogs.py:1:1: D100 Missing docstring in public module

C:  1, 0: Missing module docstring (missing-docstring)
ERROR: InvocationError: '/home/travis/build/home-assistant/home-assistant/.tox/pylint/bin/pylint homeassistant'
```

Other components didn't seem to do this when importing asyncio

--- a/.coveragerc
+++ b/.coveragerc
@@ -496,6 +496,7 @@ omit =
     homeassistant/components/sensor/deluge.py
     homeassistant/components/sensor/deutsche_bahn.py
     homeassistant/components/sensor/dht.py
+    homeassistant/components/sensor/discogs.py
     homeassistant/components/sensor/dnsip.py
     homeassistant/components/sensor/dovado.py
     homeassistant/components/sensor/dte_energy_bridge.py

--- a/homeassistant/components/sensor/discogs.py
+++ b/homeassistant/components/sensor/discogs.py
@@ -1,3 +1,9 @@
+"""
+Shows the amount of records in a user's Discogs collection.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.discogs/
+"""
 import asyncio
 import logging
 from datetime import timedelta
@@ -19,7 +25,7 @@ PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
 
 REQUIREMENTS = ['discogs_client==2.2.1']
 
-SCAN_INTERVAL = timedelta(seconds=7200)
+SCAN_INTERVAL = timedelta(hours=2)
 
 
 @asyncio.coroutine

--- a/homeassistant/components/sensor/discogs.py
+++ b/homeassistant/components/sensor/discogs.py
@@ -14,6 +14,7 @@ import homeassistant.helpers.config_validation as cv
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import (CONF_TOKEN)
 from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.aiohttp_client import SERVER_SOFTWARE
 
 _LOGGER = logging.getLogger(__name__)
 
@@ -43,8 +44,7 @@ class DiscogsCollection(Entity):
     def __init__(self, token):
         """Initialize the sensor."""
         import discogs_client
-        discogs = discogs_client.Client(
-            'HomeAssistantDiscogs/0.1.0', user_token=token)
+        discogs = discogs_client.Client(SERVER_SOFTWARE, user_token=token)
 
         self._ds_user = discogs.identity()
         self._state = None

